ci/test_unit.sh

```diff
@@ -17,4 +17,7 @@
 #!/bin/bash
 set -e
 
-TF_GPU_ALLOCATOR=cuda_malloc_async python -m pytest -rxs tests/unit
\ No newline at end of file
+TF_GPU_ALLOCATOR=cuda_malloc_async python -m pytest -rxs tests/unit
+
+# Only run tests marked with `horovod` with `horovodrun` command
+horovodrun -np 2 sh ./examples/usecases/multi-gpu/hvd_wrapper.sh python -m pytest -rxs tests/unit -m horovod
```

Comment:
If my comment below is possible, would it be better to run this inside the context of the regular test/unit call in line 20?

Reply:
Tried the `subprocess` approach but was unsuccessful as mentioned in my response above.